charts/internal/shoot-control-plane/templates/firewall/firewall.yaml

```diff
@@ -1,15 +0,0 @@
----
-apiVersion: v1
```

Comment:
My bad: This namespace definition has to be kept as otherwise helm will delete the namespace. 🙈

Reply:
Will fix

Reply:
Done, after 3 tries :-)